nixos/modules/services/web-apps/keycloak.nix

```diff
@@ -272,6 +272,21 @@ in
         '';
       };
 
+      realmFiles = mkOption {
+        type = attrsOf str;
```

Comment:
wouldn't be list of paths more intuitive? was kind of surprised that i have to pass strings to it. using file paths could be also good for secrets so they won't get imported into the store

Reply:
You`re absolutely right. I'm not even sure why this was an attrSet in the first place. Changed it to accept a list of paths now.